charts/fission-all/values.yaml

```diff
@@ -308,6 +308,11 @@ canaryDeployment:
 
 ## Message Queue Trigger Kind, KEDA: enable and configuration
 mqt_keda:
-  enabled: true
+  enabled: false
   connector_images:
-    kafka: fission/keda-kafka
+    kafka:
+      image: fission/keda-kafka
```

Comment:
Probably call it fission/keda-kafka-http-connector? (LMK if you think that's too big a name though) So there is no ambiguity?

Reply:
Done